ui/components/multichain/pages/connections/connections.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Connections } from './connections';
+
+export default {
+  title: 'Components/Multichain/Connections',
+};
+export const DefaultStory = () => (
+  <div
+    style={{
+      width: '400px',
+      height: '600px',
+    }}
+  >
```

Comment:
It would be great to see this in a responsive format. Connection screens can also be full width if the browser is set to full screen.

Reply:
Hmm... can I make this happen by taking out the width but leaving the height? I want to make sure the flex alignment works in the story